target/linux/bcm27xx/modules/other.mk

```diff
@@ -55,13 +55,46 @@ endef
 $(eval $(call KernelPackage,smi-bcm2835-dev))
 
 
+define KernelPackage/rp1
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=RP1 firmware
+  KCONFIG:=CONFIG_FIRMWARE_RP1
+  FILES:=$(LINUX_DIR)/drivers/firmware/rp1.ko
+  AUTOLOAD:=$(call AutoLoad,21,rp1)
+  DEPENDS:=@TARGET_bcm27xx_bcm2712
+endef
+
+define KernelPackage/rp1/description
+  This driver provides a firmware interface to the RP1 processor using shared
+  memory and a mailbox.
+endef
+
+$(eval $(call KernelPackage,rp1))
+
+
+define KernelPackage/rp1-pio
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=RP1 PIO block support
+  KCONFIG:=CONFIG_RP1_PIO
+  FILES:=$(LINUX_DIR)/drivers/misc/rp1-pio.ko
+  AUTOLOAD:=$(call AutoLoad,21,rp1-pio)
+  DEPENDS:=@TARGET_bcm27xx_bcm2712 +kmod-rp1
+endef
+
+define KernelPackage/rp1-pio/description
+  Driver providing control of the Raspberry Pi PIO block, as found in RP1
+endef
+
+$(eval $(call KernelPackage,rp1-pio))
+
```

Comment:
This is added two times, see other entry at line 144.

Reply:
Thanks hauke, will send a new PR to correct.